templates/ep19/bs/talk_voting/voting.html

```diff
@@ -1,5 +1,7 @@
 {% extends "ep19/bs/base.html" %}
 
+{% load cms_tags %}
```

Comment:
why cms_tags?

Reply:
Good catch - I was using `page_url` before I realised it didn't work :sweat_smile: removing!